src/runtime/components/_base.ts

```diff
@@ -109,3 +118,47 @@ export const useBaseImage = (props: ExtractPropTypes<typeof baseImageProps>) =>
     modifiers
   }
 }
+
+export const useBasePicture = (props: ExtractPropTypes<typeof basePictureProps>) => {
```

Comment:
For simplicity and reduced bundle size, do you think we can avoid splitting out `useBasePicture` and instead override the props directly in `nuxt-img`?